### ISSUE TITLE
Missing favicon.png in snorkel-lite

### ISSUE THREAD

**Issue by alfa07** [logv/snorkel#47]
In pristine `pip install snorkel-lite` I am getting the following errors in `snorkel.frontend`:
```
[2020-03-26 12:21:12,314] ERROR in app: Exception on /favicon.ico [GET]
Traceback (most recent call last):
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/snorkel/web.py", line 87, in get_favicon
    with open(os.path.join(app.static_folder, "favicon.png"), "rb") as f:
IOError: [Errno 2] No such file or directory: '/data/home/msokolov/dev/snorkel-db/penv/lib/python2.7/site-packages/snorkel/static/favicon.png'
```

**Comment by okayzed** [comment]
Thanks for reporting! technically, this has not much effect other than cluttering the logs since its just causing a 500 on the favicon.ico request. I will fix anyways though

**Comment by okayzed** [comment]
Fixed in 0.0.29 to return an empty icon if not found. Can investigate how to make the static folder more accessible in the future
